README.md

```diff
@@ -200,26 +201,36 @@ Adding brokers to the cluster
 ```
 # ./kafka-mesos.sh help add
 Add broker
-Usage: add <id-expr>
-
-id-expr examples:
-  0      - broker 0
-  0,1    - brokers 0,1
-  0..2   - brokers 0,1,2
-  0,1..2 - brokers 0,1,2
-  *      - any broker
+Usage: add <id-expr> [options]
 
 Option              Description
 ------              -----------
---attributes        slave attributes (rack:1;role:master)
+--constraints       constraints (hostname=like:master,
+                      rack=like:1.*)
 --cpus <Double>     cpu amount (0.5, 1, 2)
 --failoverDelay     failover delay (10s, 5m, 3h)
 --failoverMaxDelay  max failover delay. See failoverDelay.
 --failoverMaxTries  max failover tries
 --heap <Long>       heap amount in Mb
---host              slave hostname
 --mem <Long>        mem amount in Mb
---options           kafka options (a=1;b=2)
+--options           kafka options (log.dirs=/tmp/kafka/$id,
+                      num.io.threads=16)
+id-expr examples:
+  0      - broker 0
+  0,1    - brokers 0,1
+  0..2   - brokers 0,1,2
+  0,1..2 - brokers 0,1,2
+  *      - any broker
+
+constraint examples:
+  like:master     - value equals 'master'
```

Comment:
Usually "like" doesn't means exact match, but partial matching (http://www.w3schools.com/sql/sql_like.asp).
Why not just support hostname=master and hostname!=master conditions? instead of like and unlike